src/api/models.py

```diff
@@ -4,6 +4,7 @@
 from sqlalchemy.sql import func
 
 db = SQLAlchemy()
```

Comment:
ah, buena pregunt. quizá podemos probar https://realpython.com/flask-by-example-part-2-postgres-sqlalchemy-and-alembic/